### PR TITLE
Add ARA suport


### PR DESCRIPTION
**Pull request by evrardjp**
This makes instrumentation easier

### CODE REVIEW

**Comment by evrardjp** [comment]
Do you think we should add a test in install_ansible venv to force installing ara when not present?

**Comment by nicolasbock** [comment]
You mean as in upgrading an existing environment? That would be helpful, yes.

**Comment by Itxaka** [comment]
> Do you think we should add a test in install_ansible venv to force installing ara when not present?

we should probably decide if we just want to run everything of an virtualenv or let the package installation to the end user (In my case I use pipenv for dependency management so I rather have control over it myself) with proper instructions in the readme

Also ara looks really nice! Does it introduce any delays in the playbook execution? In that case, should we make ARA optional with a env/config switch?

**Comment by evrardjp** [comment]
@Itxaka it's very lightweight.

I think we can add more conditionals in the wrapper script, but it would make the wrapper slower.

For the BYO ansible, I am fine with that.

File: script_library/run-ansible.sh [hunk 1]
```diff
@@ -35,6 +35,9 @@ function run_ansible(){
         echo "Inventory directory (${inventorydir}) exists, adding it to the ansible-playbook call."
         inventory="-i ${inventorydir}"
     fi
+    if [[ -f ${HOME}/.socok8svenv/ara.rc ]]; then
+        source ${HOME}/.socok8svenv/ara.rc
```
Comment (Itxaka) [inline script_library/run-ansible.sh]: can we add an `ECHO` statement here so we are explicit that we are loading it?

Comment (evrardjp) [reply script_library/run-ansible.sh]: Done.

**Pull request merged**